extras/CMakeLists.txt

```diff
@@ -21,6 +16,11 @@ endfunction()
 add_library(mapbox-base-extras INTERFACE)
 add_library(Mapbox::Base::Extras ALIAS mapbox-base-extras)
 
+execute_process(
```

Comment:
This is to avoid mapbox/io from attempting to fetch its non-existent git submodule.